test/@types/vitest.d.ts

```diff
@@ -38,6 +39,18 @@ declare module "vitest" {
      */
     toEqualArrayUnsorted(expected: T[]): void;
 
+    // # endregion Generic Matchers
+
+    // # region GameManager Matchers
```

Comment:
```suggestion
    // #endregion Generic Matchers

    // #region GameManager Matchers
```

Shouldn't have a space between the `hash` and the name; it doesn't work on all editors if you do, iirc.